hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/HermesConsumers.java

```diff
@@ -64,6 +65,9 @@ public static void main(String... args) {
         assignmentCache = serviceLocator.getService(SubscriptionAssignmentCache.class);
         oAuthHttpClient = serviceLocator.getService(OAuthClient.class);
 
+        HttpClientsWorkloadReporter httpClientsWorkloadReporter = serviceLocator.getService(HttpClientsWorkloadReporter.class);
+        httpClientsWorkloadReporter.registerMetrics();
```

Comment:
this could be called in `start()`

Reply:
and could be renamed to `start` as well :)

Reply:
Done